### DISCUSSION TITLE
[Feature] secondary classification

### DISCUSSION THREAD

**Discussion by 849025143** [immich-app/immich#12689] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add another album to the existing one.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by radh21301** [comment]
What do you mean?

**Comment by bo0tzz** [comment]
This FR is both too vague and probably a duplicate, so I'll close it.
